components/datadog/apps/fakeintake/fargateFakeintakeService.go

```diff
@@ -300,9 +300,8 @@ func fargateServiceFakeIntakeWithLoadBalancer(e aws.Environment, name string, na
 			SecurityGroups: pulumi.ToStringArray(e.DefaultSecurityGroups()),
 			Subnets:        e.RandomSubnets(),
 		},
-		LoadBalancers:             balancerArray,
-		TaskDefinition:            taskDef.TaskDefinition.Arn(),
-		ContinueBeforeSteadyState: pulumi.BoolPtr(true),
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review. Do look for something to sincerely praise. -->
**👏 praise**
😲